paddle/fluid/framework/parallel_executor.cc

```diff
@@ -336,7 +344,8 @@ ParallelExecutor::ParallelExecutor(const std::vector<platform::Place> &places,
 
   if (member_->use_cuda_) {
 // Bcast Parameters to all GPUs
-#if defined(PADDLE_WITH_CUDA) && !defined(_WIN32)
+#if defined(PADDLE_WITH_CUDA)
+#if !defined(_WIN32)
```

Comment:
There should not have so many `if`, it is ugly.

Reply:
Cleaned.